elfo-logger/src/line_buffer.rs

```diff
@@ -0,0 +1,274 @@
+use std::mem;
+
+use crate::line_transaction::Line;
+
+// BufferConfiguration
```

Comment:
`BufferConfig`?

Also, everywhere in elfo sections are formatted as `// === Name ===`, but it's not required, so you can resolve this comment